cli/sync_service/object.go

```diff
@@ -353,13 +353,18 @@ func ObjectPublish(org string, userPw string, objType string, objId string, objP
 		msgPrinter.Printf("Digital sign with %s will be performed for data integrity. It will delay the MMS object publish.\n", hashAlgorithm)
 
 		// Create public key. Sign data. Set "hashAlgorithm", "publicKey" and "signature" field
-		if publicKey, signature, err := signObjData(objFile, hashAlgorithm, dsHash, privKeyFilePath); err != nil {
+		file, err := os.Open(objFile)
+		if err != nil {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("unable to open object file %v: %v", objFile, err))
+		}
+		if publicKey, signature, err := SignObjData(file, hashAlgorithm, dsHash, privKeyFilePath); err != nil {
```

Comment:
should we close the flie `file.Close()` here before giving back error?

Reply:
Changed to a `defer file.Close()` statement